tasks/e2e.sh

```diff
@@ -175,25 +197,46 @@ npm link $root_path/packages/eslint-config-react-app
 npm link $root_path/packages/react-dev-utils
 npm link $root_path/packages/react-scripts
 
+# ...and we need  to remove template's .babelrc
```

Comment:
Because it's using `mocha`, that needs it.
I could move its config into the template's `package.json`, if we decide on using it (view [this](#discussion_r95970301) previous comment)